### PR TITLE
Various fixes and improvements

### PR DESCRIPTION
**Pull request by gnapse**
## Short description

Fixes and improvements for issues arising from our first attempts at using v10.0.0-beta.4 in twist-web.

Check `CHANGELOG.md` for details.

## PR Checklist

-   [x] Added tests for bugs / new features
-   [x] Updated docs (storybooks, readme)
-   [x] Executed `npm run validate` and made sure no errors / warnings were shown
-   [x] Described changes in `CHANGELOG.md`
-   [x] Bumped version in `package.json` and `package-lock.json` (`npm --no-git-tag-version version <major|minor|patch>`) [ref](https://docs.npmjs.com/cli/v6/commands/npm-version)
-   [ ] Updated all static build artifacts (`npm run build-all`)

## Versioning

To be released as a new beta patch version.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 
+## v10.0.0-beta.5
+
+-   [Breaking] The various `Menu*` components props no longer consider `className` as a valid prop.
+-   [Breaking] The `Stack` component's `align` prop type is now `'start' | 'center' | 'end'` instead
+    of `'left' | 'center' | 'right'`.
+-   [Fix] Colour of quaternary button is now correct.
+-   [Docs] Improvements to the storybook examples for the `Menu` components.
+-   [Fix] Modal focus lock is now working correctly.
+-   [Fix] Modals are now aligned to the top of the viewport
```
Comment (craigcarlyle) [inline CHANGELOG.md:13]:
```suggestion
-   [Fix] Modals are now aligned to the top of the viewport.
```

File: src/components/menu/menu.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+            <Column width="auto">{name}</Column>
+            <Column width="content">
+                <Menu onItemSelect={(itemValue) => action(`${String(itemValue)} '${name}'`)(value)}>
+                    <MenuButton
+                        as={Button}
+                        variant="quaternary"
+                        icon={<OverflowIcon />}
+                        aria-label={ariaLabel}
+                    />
+                    <MenuList aria-label={ariaLabel}>
```
Comment (craigcarlyle) [inline src/components/menu/menu.stories.tsx:194]: Should the `aria-label` be the same for both the `MenuButton` and `MenuList`?

**Review by craigcarlyle — APPROVED** [review]
🎉

File: src/components/menu/menu.stories.tsx (hunk 2 above)
Comment (gnapse) [reply src/components/menu/menu.stories.tsx:194]: This is just for the storybook example. Is up to the consumer of these components to provide meaningful labels for each part.

**Pull request merged**